spec/acceptance/ferm_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper_acceptance'
+
+describe 'ferm' do
+  context 'install ferm' do
+    pp = %(
+    class { 'ferm':
```

Comment:
i believe you're missing `Boolean $manage_initfile` which needs to be set to `true` in order to make the acceptance tests actually find the `ferm` service.

Reply:
`manage_initfile` is only required for CentOS 6. CentOS 6 doesn't ship a init file, so this module can place a sysv init script there.

Reply:
probably some OSes need it.